src/XMakeCommandLine/UnitTests/XMake_Tests.cs

```diff
@@ -633,7 +632,8 @@ public void ConfigurationInvalid()
                     "</Project>";
                 File.WriteAllText(pathToProjectFile, projectString);
 
-                output = RunProcessAndGetOutput(newPathToMSBuildExe, "\"" + pathToProjectFile + "\"", expectSuccess: false);
+                var msbuildParameters = "\"" + pathToProjectFile + "\"";
+                output = runMSBuildWithinCoreRun(newPathToMSBuildExe, msbuildParameters, expectSuccess: false);
```

Comment:
@AndyGerlicher I like your idea better than my delegate one. I was just doing the delegate one and all the new names made it confusing.